src/platforms/javascript/guides/cordova/ionic.mdx

```diff
@@ -1,11 +1,19 @@
 ---
 title: Ionic
 sidebar_order: 201
+noindex: true
+notoc: true
```

Comment:
For some reason this page is still showing up in the TOC in the preview. I'm going to pull this branch into my local and test it.